lib/node_modules/@stdlib/blas/ext/base/dsumkbn/benchmark/benchmark.js

```diff
@@ -21,14 +21,19 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require('@stdlib/random/base/uniform');
```

Comment:
Please conform to project spacing conventions. You should be consistent with what you see.